lib/logger.js

```diff
@@ -0,0 +1,46 @@
+const colors = require('colors')
+
+function log(str, tags = []) {
```

Comment:
This is wonderful. The console looks beautiful, and you know that I mean it when I saw I appreciate a beautiful CLI.